### DISCUSSION TITLE
Exported attachments in JPEG when using "-c efficient"

### DISCUSSION THREAD

**Discussion by jsonmar** [ReagentX/imessage-exporter#337] [Q&A] [answered]
Using an iPhone unencrypted back up. I was expecting **all** of the images in the attachments folder to be preserved in the .heic format. 

Is this not the case when using efficient?

Thank you

**Comment by ReagentX** [comment, marked as answer]
`-c efficient` does not perform any conversion to attachment files:

https://github.com/ReagentX/imessage-exporter/blob/f88e1360f6f84b112f0a422e35560f2b6ddeb4a7/imessage-exporter/src/app/attachment_manager.rs#L92

https://github.com/ReagentX/imessage-exporter/blob/f88e1360f6f84b112f0a422e35560f2b6ddeb4a7/imessage-exporter/src/app/attachment_manager.rs#L116-L129

If the exported files are `JPEG`, they were already `JPEG` in the backup.
